docs/source/release_notes/release-notes.rst

```diff
@@ -51,6 +51,11 @@ develop
          - Change
 
     *    - |improved|
+         - Schema mutations against the Cassandra KVS are now HA.
+           Previously, some Cassandra KVS required that after some schema mutations all cassandra nodes must agree on the schema version.
+           Now, only a quorum of nodes must agree, if the remaining nodes are unreachable.
```

Comment:
in line with my method name re-name ask, I think the wording of "all available|reachable nodes must agree and at least a quorum of nodes must be available|reachable" is more understandable and in-line with the actual logic.

Reply:
Yep, I think we should still update this to @tpetracca's wording